langlib/lang.runtime/src/main/ballerina/runtime.bal

```diff
@@ -94,7 +94,7 @@ isolated function externGetStackTrace() returns CallStackElement[] = @java:Metho
     'class: "org.ballerinalang.langlib.runtime.GetStackTrace"
 } external;
 
-# Type of the function passed to `onGracefulStop.
+# Type of the function passed to `onGracefulStop`.
```

Comment:
@gabilang Maybe this could be be the reason why API docs were not generated for these two. Sent a PR to the ballerina-spec also https://github.com/ballerina-platform/ballerina-spec/pull/1180.